langchain_tools_demo/main.py

```diff
@@ -51,10 +52,11 @@ async def lifespan(app: FastAPI):
 BASE_HISTORY = [{"role": "assistant", "content": "How can I help you?"}]
 
 
-@app.get("/", response_class=HTMLResponse)
+@app.route("/", methods=["GET", "POST"])
```

Comment:
Why is this sending POST here?

Reply:
Changed to GET after rebasing on the reset PR.